src/Symfony/Component/Workflow/Event/Event.php

```diff
@@ -43,6 +48,9 @@ public function getMarking()
         return $this->marking;
     }
 
+    /**
+     * @return TSubject|object
```

Comment:
This does not look correct. We should always get a `TSubject` here. Wasn't that the whole point of the PR?

Reply:
Nope, in this case psalm have issue, says that it can be object or Tsubject

Reply:
What issue? https://psalm.dev/r/4582b1218d

Reply:
I will remove it now, and we see

Reply:
@ro0NL And, here we have the issue https://github.com/symfony/symfony/runs/453131776

Reply:
Weird. Maybe try to reproduce on psalm.dev, but here psalm seems wrong AFAIK.